README.md

```diff
@@ -369,4 +370,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+
+In the 'Kubernetes' tab of the Docker settings,  click the 'Enable Kubernetes' checkbox.
+
+### 3. Increase Memory and CPUs
+
+In the Advanced tab of the Docker preferences, set 'CPUs' to 4.
+
+While Alfresco Digital Business Platform installs and runs with only 8 GiB allocated to Docker, 
+for better performance we recommend that 'Memory' value be set slightly higher, to at least 10 - 12 GiB
+(depending on the size of RAM in your workstation). 
+
+### 4. Change/Verify Context
+
+If you have previously deployed the DBP to AWS or minikube you will need to change/verify that the `docker-for-desktop` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context if needed
+```
+
+### 5. Restart docker  
+
+Docker can be faulty on its first start. So, it is always safer to restart it before proceeding. Right click on the docker icon in the system tray, then left click "restart...". 
+
+### 6. Install helm
+
+Install chocolatery using command prompt (with admin rights). 
+
+```bash
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Initialize Helm Tiller (Server Component)
+
+```bash
+helm init
+```
+
+```bash
+choco install kubernetes-helm
+```
+
+### 7. Create your namespace
+
+```bash
+kubectl create namespace <yourNamespace>
+```
+
+### 8. Pull secrets
+
+Log in to quay.io
+```bash
+docker login quay.io
+```
+Give username and password if prompted.
+
+Generate a base64 value for your dockercfg, this will allow Kubernetes to access quay.io
+
+```bash
+certutil -encode "%USERPROFILE%\.docker\config.json" tmp.b64 && findstr /v /c:- tmp.b64
+```
+Copy the string output.
+
+Create the file secrets.yaml. And insert the copied string into the file where specified:
+
+```bash
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-registry-secret
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: <replace this with the string>
+```
+
+Note that when you paste the string output in the data section, it may be pasted with new lines in it. So, make sure to take out the new lines. And leave a single space between "data:" and the string. 
+
+Create the secret in your namespace. 
+
+```bash
+kubectl create -f secrets.yaml --namespace <yourNamespace>
+```
+
+### 9. Download the alfresco helm chart
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 10. Authorize connections
+
+Check that the config.json file in C:\Users\<YourUserName>\.docker has a string after the word "auth".
+
+Go to the config.json file in "C:\Users\Ayman Harake\.docker", and check that there is a string after auth, such as in the following example.
```

Comment:
I have added this step, however, it may not work for everyone:

Go to the config.json file in "C:\Users\yourUserProfile\.docker", and delete anything inside the string after credsStore to make sure that it is empty as follows:

```bash
"credsStore": ""
```